content/de/contribute/_index.md

```diff
@@ -0,0 +1,110 @@
+
+1) Neue Begriffe vorschlagen
+2) Existierende Begriffe aktualisieren 
+3) Bei der Übersetzung helfen
+
+## Werde Mitglied der Glossar-Community!
+Wer regelmäßig einen Beitrag leisten möchten, sollte an den monatlichen Treffen der Glossar-Arbeitsgruppe teilnehmen. Details zu den Treffen befinden sich im CNCF-Kalender. Ihr könnt euch auch mit den Betreuern und anderen Mitwirkenden in unserem #glossary Kanal über den [CNCF Kalender] (https://www.cncf.io/calendar/) austauschen - wir freuen uns auf euch! 
+
+## Neue Begriffe vorschlagen
+Ihr könnt neue Begriffe vorschlagen, an dem andere arbeiten sollen, oder selbst eine neue Definition erstellen. Der erste Schritt ist jedenfalls das Erstellen eines GitHub Issues für ein neues Anliegen (ein GitHub-Konto wird benötigt).
+
+Es folgt eine Schritt-für-Schritt-Anleitung für diejenigen, die mit GitHub noch nicht vertraut sind. **Wenn Du bereits ein GitHub-Profi** bist, verwende bitte unsere Vorlagen für die Erstellung eines neuen GitHub-Issue, achte darauf die richtigen Namenskonventionen und den [Style Guide] (https://glossary.cncf.io/style-guide/) einzuhalten, mach dich vertraut mit unseren Vorlagen und informiere uns über eingestellte Pull Requests über Slack (damit wir sie nicht übersehen). Vielen Dank! 
+
+### Neues GitHub Issue anlegen
+Gehe zu den [Github Issues](https://github.com/cncf/glossary/issues) des Glossar Repositories und klicke auf "New issue".
+
+![issues](/images/how-to/howto-01.png)
+
+Dort sind eine Vielzahl von Vorlagen zu sehen. Um einen neuen Begriff in deutscher Sprache vorzuschlagen, wähle "German Language Glossary Request".
+
+![templates](/images/how-to/howto-02.png)
+
+Füge den gewünschten Begriff ein, und klicke auf "Submit new issue". Falls du den Begriff lediglich vorschlagen wolltest, bist du jetzt bereits fertig! Falls du auch daran arbeiten möchtest, folge den nächsten Schritten. 
+
+![neues issue](/images/how-to/howto-03.png)
+
+Treten dem #glossary Kanal in dem CNCF Slack bei und teile  @Catherine Paganini, @jmo und @Seokho Son mit, dass du einen neuen Vorschlag eingereicht hast und daran arbeiten möchtest. Sie werden dir das GitHub-Issue zuweisen und allen anderen signalisieren, dass der Begriff bereits jemandem zugewiesen ist.
+
+Hier kannst du sehen, dass die ersten drei Begriffe verfügbar sind, während der letzte Begriff bereits jemandem zugewiesen wurde.
+
+![Begriff zuweisen](/images/how-to/howto-04.png)
+
+Beachte bitte, dass du jeweils nur an einen Begriff gleichzeitig arbeiten kannst. Falls du an mehreren Begriffen arbeiten möchtest, schließe bitte zuerst den einen ab, bevor du an dem anderen arbeitest. 
+
+### Begriff einreichen (Erstellen eines Pull Request)
+
+Vor dem Start bitte sicherstellen, dass der [Style Guide] (https://glossary.cncf.io/style-guide/) beachtet wird - das hilft dabei, die weiteren Interaktionen zu minimieren. Wie im Style Guide angegeben, empfehlen wir, mit einem Google- oder Word-Dokument zu beginnen. 
```

Comment:
Maybe "Vor dem Start, stell bitte sicher, dass..."?

and how about..." — das wird weitere Interaktionen minimieren."  >> without "das hilft dabei"